getStaticProps.js

```diff
@@ -54,7 +54,7 @@ export default function createStaticProps(swingsetOptions = {}) {
             scope: {
               componentProps: props
                 ? requireFromString(props, propsPath)
-                : undefined,
+                : null,
```

Comment:
Just curious - why does this make a difference?

Reply:
Based on the NextJS error I ran into - I guess `undefined` cannot be serialized as a static prop, but `null` can (?)